src/System.Net.Security/src/System/Net/SecureProtocols/NegoState.cs

```diff
@@ -483,24 +437,24 @@ private bool CheckSpn()
         //
         private void StartSendBlob(byte[] message, LazyAsyncResult lazyResult)
         {
-            Win32Exception win32exception = null;
+            Exception exception = null;
```

Comment:
I presume that now there is no reason for this type change here? The PAL will always return a `Win32Exception` on all platforms.